### PR TITLE
Drug Dealer Profession

### PR DESCRIPTION
**Pull request by slimeboy460**
#### Summary
None

#### Purpose of change

I saw a post made by ferret in discord which reminded me of drugs and drug dealing, which caused me to remember that there was no drug dealer profession.

#### Describe the solution

My memories will be disvalued with the addition of the drug dealer profession/

#### Testing

A bunch of JSON lint adjustments

### CODE REVIEW

File: data/json/professions.json [hunk 1]
```diff
@@ -3895,5 +3895,15 @@
       "male": [ "boxer_shorts" ],
       "female": [ "bra", "panties" ]
     }
+  },
+  {
+    "type": "profession",
+    "id": "local_drug_dealer",
+    "name": "Drug Dealer",
+    "description": "You were about to make your biggest sale yet, but when you met the buyer he tried to bite you. To top it off, it seems that this wasn't some sort of bad high for the user, as the whole town tried going for your throat too.",
```
Comment (LaVeyanFiend) [inline data/json/professions.json:3903]:
```suggestion
    "description": "You were about to make your biggest sale yet, but when you met the buyer he tried to bite you.  To top it off, it seems that this wasn't some sort of bad high for the user, as the whole town tried going for your throat too.",
```

File: data/json/professions.json [hunk 2]
```diff
@@ -3895,5 +3895,15 @@
...
+    "points": 2,
+    "skills": [ { "level": 1, "name": "melee" }, { "level": 2, "name": "stabbing" } ],
+    "items": { "both": [ "hoodie", "jeans", "mbag", "socks", "sneakers", "bandana", "gloves_medical", "crack", "crack", "crack", "switchblade" ], "male": [ "briefs" ], "female": [ "sports_bra", "boy_shorts" ] },
```
Comment (LaVeyanFiend) [inline data/json/professions.json:3906]: You can put the crack all in one bag, see #47254.

Comment (slimeboy460) [reply data/json/professions.json:3906]: thanks for passing that to me, I'll update it (damn, 2 drug prs in an hour)

File: data/json/professions.json [hunk 3]
```diff
@@ -3895,5 +3895,17 @@
...
+    "name": "Drug Dealer",
+    "description": "You were about to make your biggest sale yet, but when you met the buyer he tried to bite you.  To top it off, it seems that this wasn't some sort of bad high for the user, as the whole town tried going for your throat too.",
+    "points": 2,
+    "skills": [ { "level": 1, "name": "melee" }, { "level": 2, "name": "stabbing" } ],
+    "items": { "both": [ "hoodie", "jeans", "mbag", "socks", "sneakers", "bandana", "gloves_medical", "switchblade" ], "male": [ "briefs" ], "female": [ "sports_bra", "boy_shorts" ] },
+    "entries": [ { "item": "crack", "charges": 25, "container-item": "bag_zipper" } ],
+    "traits": [ "LIAR" ]
   }
 ]
+
```
Comment (LaVeyanFiend) [inline data/json/professions.json:3911]:
```suggestion
```
This is a second newline.

**Comment by slimeboy460** [comment]
Honestly, with the amount of people who use drugs in this game, I'm surprised there's not more PRs around. Maybe I'm just not seeing them, who knows?

File: data/json/professions.json [hunk 4]
```diff
@@ -3895,5 +3895,17 @@
...
       "female": [ "bra", "panties" ]
     }
+  },
+  {
+    "type": "profession",
+    "id": "local_drug_dealer",
+    "name": "Drug Dealer",
+    "description": "You were about to make your biggest sale yet, but when you met the buyer he tried to bite you.  To top it off, it seems that this wasn't some sort of bad high for the user, as the whole town tried going for your throat too.",
+    "points": 2,
+    "skills": [ { "level": 1, "name": "melee" }, { "level": 2, "name": "stabbing" } ],
```
Comment (LaVeyanFiend) [inline data/json/professions.json:3905]: I think you could add some social skill here for a salesman(of certain substances).

Comment (slimeboy460) [reply data/json/professions.json:3905]: I KNEw I MISSED SOMETHING

File: data/json/professions.json [hunk 5]
```diff
@@ -3895,5 +3895,16 @@
...
     }
+  },
+  {
+    "type": "profession",
+    "id": "local_drug_dealer",
+    "name": "Drug Dealer",
+    "description": "You were about to make your biggest sale yet, but when you met the buyer he tried to bite you.  To top it off, it seems that this wasn't some sort of bad high for the user, as the whole town tried going for your throat too.",
+    "points": 2,
+    "skills": [ { "level": 1, "name": "melee" }, { "level": 2, "name": "stabbing" }, { "level": 2, "name": "speech" } ],
+    "items": { "both": [ "hoodie", "jeans", "mbag", "socks", "sneakers", "bandana", "gloves_medical", "switchblade" ], "male": [ "briefs" ], "female": [ "sports_bra", "boy_shorts" ] },
```
Comment (BrettDong) [inline data/json/professions.json:3906]:
```suggestion
    "items": {
      "both": [ "hoodie", "jeans", "mbag", "socks", "sneakers", "bandana", "gloves_medical", "switchblade" ],
      "male": [ "briefs" ],
      "female": [ "sports_bra", "boy_shorts" ]
    },
```

File: data/json/professions.json [hunk 6]
```diff
@@ -3895,5 +3895,20 @@
...
+    "name": "Drug Dealer",
+    "description": "You were about to make your biggest sale yet, but when you met the buyer he tried to bite you.  To top it off, it seems that this wasn't some sort of bad high for the user, as the whole town tried going for your throat too.",
+    "points": 2,
+    "skills": [ { "level": 1, "name": "melee" }, { "level": 2, "name": "stabbing" }, { "level": 2, "name": "speech" } ],
+    "items": {
+      "both": [ "hoodie", "jeans", "mbag", "socks", "sneakers", "bandana", "gloves_medical", "switchblade" ],
+      "male": [ "briefs" ],
+      "female": [ "sports_bra", "boy_shorts" ]
+    },
+    "entries": [ { "item": "crack", "charges": 25, "container-item": "bag_zipper" } ],
```
Comment (ZhilkinSerg) [inline data/json/professions.json:3911]:
```suggestion
```

File: data/json/professions.json [hunk 7]
```diff
@@ -3895,5 +3895,20 @@
...
+  },
+  {
+    "type": "profession",
+    "id": "local_drug_dealer",
+    "name": "Drug Dealer",
+    "description": "You were about to make your biggest sale yet, but when you met the buyer he tried to bite you.  To top it off, it seems that this wasn't some sort of bad high for the user, as the whole town tried going for your throat too.",
+    "points": 2,
+    "skills": [ { "level": 1, "name": "melee" }, { "level": 2, "name": "stabbing" }, { "level": 2, "name": "speech" } ],
+    "items": {
+      "both": [ "hoodie", "jeans", "mbag", "socks", "sneakers", "bandana", "gloves_medical", "switchblade" ],
```
Comment (ZhilkinSerg) [inline data/json/professions.json:3907]:
```suggestion
      "both": [ "hoodie", "jeans", "mbag", "socks", "sneakers", "bandana", "gloves_medical", "switchblade" ],
      "entries": [ { "item": "crack", "charges": 25, "container-item": "bag_zipper" } ],
```

**Pull request merged**